src/shared/components/palette/utils/first-error.pipe.ts

```diff
@@ -12,10 +12,10 @@ export class FirstErrorPipe implements PipeTransform {
     }
 
     if (!args) {
-      args = 'Field';
```

Comment:
Was this file even in conflict? Everything in here is a regression yet there's nothing _new_! 🤯